src/ocean/util/log/Logger.d

```diff
@@ -463,6 +474,24 @@ public final class Logger : ILogger
         return this.host_.context.enabled(this.level_, level);
     }
 
+    /***************************************************************************
+
+        Append a message with a severity of `Level.Debug`.
+
+        Params:
+            Args = Auto-deduced format string arguments
+            fmt = Format string to use.
+                  See `ocean.text.convert.Formatter` documentation for
+                  more informations.
+            args = Arguments to format according to `fmt`.
+
+    ***************************************************************************/
+
+    public void dbg (Args...) (cstring fmt, Args args)
```

Comment:
you could mention in documentation for the function that it is named `dbg` instead `debug` (which would be preferable) as the latter is reserved word in D.

Reply:
Fixed